packages/assets-controllers/src/NftController.test.ts

```diff
@@ -80,21 +80,24 @@ function setupController({
       allowedActions: [],
     });
   const preferences = new PreferencesController();
-  const network = new NetworkController({
-    messenger,
-    infuraProjectId: 'potato',
-  });
+  const onNetworkStateChangeCallbacks: any[] = [];
+  const changeNetwork = (providerConfig: ProviderConfig) => {
+    onNetworkStateChangeCallbacks.forEach(
+      (callback) => callback({ providerConfig }), // eslint-disable-line node/no-callback-literal
```

Comment:
Nit: This ESLint warning is here because this isn't a callback, it's an event listener. This rule is expecting parameters called `callback` to be treated like a callback in that the first parameter is always an `Error` or `undefined`/`null`.

This can be worked around by renaming this parameter to `listener` instead for example. And similarly, the collection can be called `onNetworkStateChangeListeners`